creation/web_base/frontend/singularity_script_example.sh

```diff
@@ -54,8 +54,13 @@ warn_raw() {
 [[ -z "$glidein_config" ]] && [[ -e "$GWMS_THIS_SCRIPT_DIR/../glidein_config" ]] &&
     glidein_config="$GWMS_THIS_SCRIPT_DIR/../glidein_config"
 
+# import add_config_line function
+add_config_line_source=$(grep -m1 '^ADD_CONFIG_LINE_SOURCE ' "$glidein_config" | awk '{print $2}')
+# shellcheck source=./add_config_line.source
+. "$add_config_line_source"
+
 # error_gen defined in singularity_lib.sh
-[[ -e "$glidein_config" ]] && error_gen=$(grep '^ERROR_GEN_PATH ' "$glidein_config" | cut -d ' ' -f 2-)
+[[ -e "$glidein_config" ]] && error_gen=$(gconfig_get ERROR_GEN_PATH "$glidein_config")
```

Comment:
This script may be invoked w/o passing glidein_config.
I would keep the grep for error_gen.
Alternatively, you have to protect the add_config_line lines (e.g. in an ` if [[ -e "$glidein_config" ]]...`)

Reply:
Thanks for sharing the suggestion; this has been reverted to use `grep` which was initially the case.